src/main.cpp

```diff
@@ -3136,7 +3136,7 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
         }
     }
     if (nSizeLimit <= 0)
-        nSizeLimit = DEFAULT_BLOCK_ACCEPT_SIZE * 1E5;
+        nSizeLimit = static_cast<int32_t>(DEFAULT_BLOCK_ACCEPT_SIZE * 10) * 1E5;
```

Comment:
Should probably leave a comment to indicate that the decimal truncation is intentional. The commit message could be confusing because you're still doing a float operation at the end -- `1E5` is a float literal.

For consistency maybe `std::int32_t` or change `std::uint32_t` to just `uint32_t`.